packages/x-date-pickers/src/AdapterDayjs/AdapterDayjs.ts

```diff
@@ -657,6 +660,10 @@ export class AdapterDayjs implements MuiPickersAdapter<Dayjs, string> {
     return value.week();
   };
 
+  public getDayOfWeek(value: Dayjs): number {
```

Comment:
Used to set proper (i.e. '2' for `Tuesday` on a locale where the week starts on Monday.) `aria-valuenow` on the `weekDay` token.